docs/content/Installation/operator-install.md

```diff
@@ -120,8 +120,7 @@ Here are some common examples of configuration changes most installers would mak
 Inside `conf/postgres-operator/pgo.yaml` there are various storage configurations defined.  
 
     PrimaryStorage: gce
-    XlogStorage: gce
-    ArchiveStorage: gce
+    WALStorage: gce
```

Comment:
Hm, but isn't this not actually defined? It appears to not be in the defaults that I've seen are empty.

It seems if we specify this, then one is automatically opted into using the WALStorage feature:

```go
		if apiserver.Pgo.WALStorage != "" {
			spec.WALStorage, _ = apiserver.Pgo.GetStorageSpec(apiserver.Pgo.WALStorage)
		}
```

Reply:
> Hm, but isn't this not actually defined? It appears to not be in the defaults that I've seen are empty.

Yes? As committed, `WALStorage` is present and blank `pgo.yaml`.

> It seems if we specify this, then one is automatically opted into using the WALStorage feature:
> 
> ```go
> 		if apiserver.Pgo.WALStorage != "" {
> 			spec.WALStorage, _ = apiserver.Pgo.GetStorageSpec(apiserver.Pgo.WALStorage)
> 		}
> ```

Yes. This is why the shipped pgo.yaml configuration is blank. Setting it there applies to all newly created clusters.

When it is blank in pgo.yaml and the user wants a WAL volume, they specify a storage configuration during cluster create, similar to a tablespace.